tools/2d_auto_threshold/auto_threshold.xml

```diff
@@ -96,46 +111,51 @@
        <data format="tiff" name="output" from_work_dir="out.tiff" />
     </outputs>
     <tests>
-        <!-- Tests for single-channel images -->
+        <!-- Tests for single-channel images (TIFF) -->
         <test>
             <param name="input" value="sample.tiff"/>
             <param name="method_id" value="loc_gaussian"/>

```

Comment:
```suggestion
            <conditional name="th_method">
                <param name="method_id" value="loc_gaussian"/>
            </conditional>

```

Linting is now stricter ... we need this explicitly mentioned.

GLS has a function to fix that automatically ...